src/main/kotlin/uk/gov/justice/digital/hmpps/calculatereleasedatesapi/service/CalculationService.kt

```diff
@@ -59,9 +59,6 @@ class CalculationService(
       calculationOutcomeRepository.save(transform(calculationRequest, it.key, it.value))
     }
 
-    if (calculationStatus == CONFIRMED) {
-      domainEventPublisher.publishReleaseDateChange(calculationRequest.prisonerId, calculationRequest.bookingId)
-    }
     return bookingCalculation
```

Comment:
Ive moved the publishing of the event up one level - as we have a specific endpoint for confirming a calculation. Moving it up one level means we dont need the if condition